server/data/data-jmap/src/main/java/org/apache/james/jmap/api/projections/MessageFastViewProjection.java

```diff
@@ -17,16 +17,16 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.jmap.api.preview;
+package org.apache.james.jmap.api.projections;
 
 import org.apache.james.mailbox.model.MessageId;
 import org.reactivestreams.Publisher;
 
-public interface MessagePreviewStore {
+public interface MessageFastViewProjection {
```

Comment:
I prefer `MessageFastViewPropertiesStore`, I don't think Projection is the right term

Reply:
The content of your `Store` is a projection of the actual mailbox message content.

`Projection` is thus a more specific term.

Moreover `building a projection` is a common NOSQL wording.

More common, more specific, then why being reluctant adopting it?